src/app/framework-modules/elements/ct-field-generator/component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input } from '@angular/core';
+import { FormControl} from '@angular/forms';
+import { Form } from '../../formdata/form/form';
+import { FormData } from '../../formdata/form/formData';
+
+@Component({
+	selector: 'ct-field-generator',
+	templateUrl: './component.html',
+	styleUrls: []
+})
+
+/**
+ * Automated field Generator component
+ * @fd - Formdata
+ */
+export class CTFieldGenerator {
+	@Input() public fd: FormData;

```

Comment:
1. Re. "Automated field Generator component": I have no idea what that means 😅, but maybe someone using your framework would know because they'd have the context?  If not, could we put a better description in here?  (Also, 3 adjectives for 1 noun is hard to parse; I would at least reword this to reorder the words:)
2. What is "form data"?  User input from an html form?
3. `fd` is not very self documenting.  At least name it `formData`, unless there is a better name.
4. The name of the class starts with the prefix `CT`.  Is this to namespace it?  If so, I thought in Java packaging is the mechanism for namespacing.  What does `CT` stand for?